test/components/views/settings/tabs/room/EmoteSettingsTab-test.tsx

```diff
@@ -354,4 +358,60 @@ describe("EmoteSettingsTab", () => {
             "",
         );
     });
+
+    it("should load emotes uploaded from other clients in compatibility mode", async () => {
+        mocked(cli.getRoom).mockReturnValue(room);
+        mocked(cli.isRoomEncrypted).mockReturnValue(false);
+        // @ts-ignore - mocked doesn't support overloads properly
+        mocked(room.currentState.getStateEvents).mockImplementation((type, key) => {
+            if (key === undefined) return [] as MatrixEvent[];
+            if (type === EMOTES_STATE.name) {
+                return new MatrixEvent({
+                    sender: "@sender:server",
+                    room_id: roomId,
+                    type: EMOTES_STATE.name,
+                    state_key: "",
+                    content: {},
+                });
+            }
+            if (type === EMOTES_COMP.name) {
+                return new MatrixEvent({
+                    sender: "@sender:server",
+                    room_id: roomId,
+                    type: EMOTES_COMP.name,
+                    state_key: "",
```

Comment:
this doesn't actually find any emotes added by other clients if the clients use image packs... each image pack has a state key that is the name of the image pack. i'm not sure what clients use an empty state key, but cinny creates a new image pack by default.

Reply:
The scope of this PR is supporting the required parts of MSC2545. Image pack support can be added down the line. As a quick fix, we can change the state key to "Element Compatible Emotes" instead of having a blank one.